workshop/lab1.1_welcome.md

```diff
@@ -22,4 +22,4 @@ Red Hat OpenShift Serverless is built upon the Red Hat OpenShift platform, which
 
 Red Hat OpenShift makes it easy to take advantage of delivering serverless applications. Through the CLI or through the GUI, you can easily specify which containers should be ran as serverless applications.  It does this by leveraging the open source project known as Knative.  This project provides a vendor agnostic way of making containers serverless capable.  It does this by applying two key concepts, serving and eventing to be able to create dynamically scaling, loosely coupled event driven applications.
 
-Red Hat OpenShift Serverless not only opens serverless architecture, but also adds monitoring, metering, A/B testing, canary releases, and publicly exposed routes.  Not only that, but another benefit of it running within the OpenShift ecosystem is that all of the hybrid cloud and cloud native development tools are available and all integrate and work well with each other.  This includes things like CodeReady Workspaces, supported runtimes for of your container, and thousands of integration connectors ready to go out of the box like Kafka event streams.
+Red Hat OpenShift Serverless not only opens serverless architecture, but also adds monitoring, metering, A/B testing, canary releases, and publicly exposed routes.  Not only that, but another benefit of it running within the OpenShift ecosystem is that all of the hybrid cloud and cloud native development tools are available and all integrate and work well with each other.  This includes things like CodeReady Workspaces, supported runtimes for of your container, and hundreds of integration connectors ready to go out of the box like Kafka event streams.
```

Comment:
"runtimes for of your container"

Reply:
Suggest changing the second sentence to remove the "Not only that" with a "And another benefit..."

Reply:
done